src/templates/base/2019/base.html

```diff
@@ -104,7 +107,19 @@
   <header class="alt-bg">
     <div class="container">
       <div class="top-header">
-        <a class="navigation-logo" href="{{ url_for('home', year=year, lang=lang) }}"><img src="/static/images/logo.svg" alt="{{ self.organization() }}" height="39" width="202"></a>
+        <a class="navigation-logo" href="{{ url_for('home', year=year, lang=lang) }}">
+          {% if language.lang_code != 'ja' %}
+            <span class="wa">{{ self.web_almanac() }}</span>
+            <span class="line-group">  
+              <span class="pre">{{ self.preposition() }}</span>
+              <span class="ha">{{ self.http_archive() }}</span>
+            </span>
+          {% else %}
+            <span class="ha line-group">{{ self.http_archive() }}</span>            
+            <span class="pre">{{ self.preposition() }}</span>
+            <span class="wa">{{ self.web_almanac() }}</span>                        
+          {% endif %}
```

Comment:
This is very clever, but not loving the hard coding of the language.

What would you think about moving this to the templates? So replacing this with just:

```suggestion
          {{ self.web_almanac_logo() }}
```

And then having the following in `en/2019/base.html`:

```
{% block web_almanac_logo %}
<span class="wa">Web Almanac</span>
<span class="line-group">  
   <span class="pre">By</span>
   <span class="ha">HTTP Archive</span>
</span>
{% endblock %}
```

Yes it does put a bit of code/presentation logic into the language templates but we have a few other examples of this, gives us much greater flexibility for other languages in the future, and avoid repeating this code three times.

Reply:
I did update this as requested! ;)